manifests/g/GHannesDE/FoxisWorld/1.1/GHannesDE.FoxisWorld.installer.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: GHannesDE.FoxisWorld
+PackageVersion: 1.1
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://onedrive.live.com/download?cid=352CB5CC6AC57102&resid=352CB5CC6AC57102%2136359&authkey=AOPY1x_k-6Rf1OI

```

Comment:
```suggestion
  InstallerUrl: https://onedrive.live.com/download?cid=352CB5CC6AC57102&resid=352CB5CC6AC57102%2136359&authkey=AOPY1x_k-6Rf1OI
```
Could you please distribute this application on GitHub rather distributing it on OneDrive?

I'm also receiving an error upon opening up the application:
![image](https://user-images.githubusercontent.com/11600822/120931337-ff210b00-c6e8-11eb-8073-0da909a84d5b.png)